pkg/virt-operator/creation/components/crds.go

```diff
@@ -519,6 +519,18 @@ func NewPrometheusRuleSpec(ns string) *promv1.PrometheusRuleSpec {
 							"summary": "No leading virt-operator was detected for the last 5 min.",
 						},
 					},
+					{
+						Record: "num_of_running_virt_handlers",
+						Expr:   intstr.FromString(fmt.Sprintf("sum(up{pod=~'virt-handler-.*', namespace='%s'})", ns)),
+					},
+					{
+						Alert: "MissingVirtHandler",
+						Expr:  intstr.FromString("num_of_running_virt_handlers < num_of_allocatable_nodes"),
```

Comment:
This is a little tricky because It is valid to not run virt-handler on all allocatable nodes. The use case here is where someone may want to restrict kubevirt to only run VMs on a subset of nodes. This could be achieved by only running virt-handler on those subset of nodes. we even document this. 

https://kubevirt.io/user-guide/#/installation/installation?id=restricting-virt-handler-daemonset

What would you think about an alert that fires off if there are 0 virt-handler instances online instead? It's not perfect, but it's something at least.

Reply:
Hm.. good point. This alert covers only the default daemonset w/o any restrictions.
Maybe `virt-operator` can watch for changes to the handler daemonset object and adjust rules accordingly? It will complicate this PR a bit, but will give accurate alerts...

Reply:
Good point indeed, it did not occured to me at first. But when we got issue for virt-handler on windows nodes. It is obvious.